doc/cephadm/operations.rst

```diff
@@ -355,3 +355,65 @@ the following command on the host:
 .. code-block:: bash
 
   cephadm rm-daemon --fsid CLUSTER_ID --name SERVICE_NAME
+
+
+.. _cephadm-spec-unmanaged:
+
+Disable automatic deployment of daemons
+=======================================
+
+Cephadm supports disabling the automated deployment and removal of daemons per service. In
+this case, the CLI supports two commands that are dedicated to this mode. 
+
+In order to disable the automatic management of dameons, apply
+the :ref:`orchestrator-cli-service-spec` with ``unamanged=True``. 
```

Comment:
typo: /unamanged/unmanaged

Reply:
This is a nitpick, but I'd remove "In order" and just begin the sentence with "To disable".

Reply:
fixed